css/index.css

```diff
@@ -1,3 +1,461 @@
+    padding-left: 121px;
+    padding-top: 12px;
+}
+
+img.code {
+    margin-top: 14px;
+    margin-left: 2px;
+}
+        /* ---- HEADER-BOTTOM ends here ----- */
+/* ------------- HEADER ends here ------------- */
+
+
+hr.one {
+    margin-top: 73px;
+    border: 1px solid black;
+    margin-left: 74px;
+    width: 880px;
+}
+
+/* ------------ CONTENT starts here ------------ */
+.content {
+    width: 100%;
+    margin-left: 74px;
+    margin-top: 36px;
+    margin-right: 72px;
+}
+
+        /* ---- CONTENT-TOP starts here ---- */
+.content-top {
+    width: 85%;
+    display: flex;
+    justify-content: flex-start;
+    margin-bottom: 30px;
+}
+
+.content-top > div {
+    flex: 1;
+}
+
+.content-top > div:nth-child(2) {
+    margin-left: 9px;
+}
+
+.content-top > div > h2 {
+    margin-bottom: 16px;
+}
+
+.content-top > div > p {
+    line-height: 1.13em;
+}
+        /* ---- CONTENT-TOP ends here ---- */
+
+aside {
+    margin-left: 2px;
+    margin-bottom: 29px;
+}
+
+        /* ---- CONTENT-BOTTOM starts here ---- */
+.content-bottom {
+    width: 86%;
+    display: flex;
+    justify-content: flex-start;
+}
+
+.content-bottom > div {
+    flex: 1;
+    padding-right: 20px;
+}
+
+.content-bottom > div > h2 {
+    margin-bottom: 16px;
+}
+
+.content-bottom > div > p {
+    line-height: 1.13em;
+}
+        /* ---- CONTENT-BOTTOM ends here ---- */
+/* ------------ CONTENT ends here ------------ */
+
+hr.two {
+    margin-top: 35px;
+    border: 1px solid black;
+    margin-left: 74px;
+    width: 880px;
+}
+
+/* ---------- FOOTER starts here ---------- */
+footer {
+    width: 100%;
+    margin-left: 74px;
+    margin-top: 35px;
+}
+
+footer h2 {
+	margin-bottom: 15px;
+}
+
+.address {
+	margin-bottom: 15px;
+}
+
+.phone {
+	margin-bottom: 15px;
+}
+
+.copyright {
+    margin-top: 40px;
+    margin-left: 350px;
+}
+/* ---------- FOOTER ends here ---------- */
+
+
+
+
+/* ===================== TABLET version ======================== */
+
+@media (max-width: 800px) {
+    /* ---- HEADER ---- */
+    header {
+        margin-top: 0;
+    }
+    .header-top {
+        padding-top: 0;
+        width: 100%;
+        flex-direction: column-reverse;
+        align-items: center;
+        align-content: center;
+    }
+    .home-page {
+        width: 100%;
+        margin-right: 287px;
+        margin-top: 3px;
+    }
+    .items {
+        width: 100%;
+        margin-left: 2px;
+        padding-right: 9px;
+        margin-top: 38px;
+    }
+    .title {
+        width: 100%;
+        margin-left: 132px;
+        margin-top: 3px;
+    }
+    img.code {
+        display: none;
+    }
+    /* --- HEADER ends ---- */
+
+    hr.one {
+        margin-top: 75px;
+        margin-left: 19px;
+        width: 760px;
+    }
+    /* ---- CONTENT ---- */
+    .content {
+        margin-left: 19px;
+        margin-right: 0;
+    }
+
+    .content-top {
+        width: 93%;
+        margin-left: 10px;
+    }
+
+    .content-top > div:nth-child(2) {
```

Comment:
I would still like you to try to move away from using nth-child(number) as elements can shift due to design files (a lot of your elements will move in React) and resorting to this will leave you with a knowledge gap on how to target items down the road. If an element shifts/is moved your nth-child(number) will no longer work the way you want it to.